channels/email/settings.rst

```diff
@@ -5,101 +5,187 @@ Settings
+
+         Currently agents can't be customers within the UI.
+         While Email communication works, agents can't see their own tickets
+         (as a customer) if they don't have access to the group.
+
 Block Notifications
-   With the regex that can be defined here, you can ensure not to send any notifications to specific systems.
-   By default this especially affects typical system addresses which can't receive Emails any way.
+   With the regex that can be defined here, you can ensure not to send any
+   notifications to specific systems. By default this especially affects typical
+   system addresses which can't receive Emails any way.
 
-   The default value is: ``(mailer-daemon|postmaster|abuse|root|noreply|noreply.+?|no-reply|no-reply.+?)@.+?``
+   The default value is:
+   ``(mailer-daemon|postmaster|abuse|root|noreply|noreply.+?|no-reply|no-reply.+?)@.+?``
 
    .. _email-settings-sender-format:
 
 Sender Format: Default value ``Agent + Format Seperator + System Address Display Name``
-   This configures the display name used in the ``FROM`` header of mails Zammad sends.
+   This configures the display name used in the ``FROM`` header of mails
+   Zammad sends.
+
+   .. note::
 
-   .. note:: This does not affect Notification mails (to agents) and password reset mails. Emails that are not sent by agents (e.g. trigger based notifications) will always fallback to ``System Address Display Name`` if needed.
+      This does not affect Notification mails (to agents) and password reset
+      mails. Emails that are not sent by agents
+      (e.g. trigger based notifications) will always fallback to
+      ``System Address Display Name`` if needed.
 
    Option set to ``Agent + Format Seperator + System Address Display Name``
-      This will cause Zammad to set the ``FROM`` header to agent name and the channels display name divided by a seperator (configured below).
+      This will cause Zammad to set the ``FROM`` header to agent name and the
+      channels display name divided by a seperator (configured below).
 
       Example: ``Christopher Miller via Chrispresso Inc.``.
+
    Option set to ``System Address Display Name``
-      This will cause Zammad to always use the display name of the used channel in the ``FROM`` header.
+      This will cause Zammad to always use the display name of the used channel
+      in the ``FROM`` header.
 
       Example: ``Chrispresso Inc.``
+
+   Option set to ``Agent Name``
+      Zammad will use the agent's name which is very personal.
+
+      .. tip::
+
+         | Usually you'd also want to remove the ticket slug from the subject
+           in those cases.
+         | Learn more in :doc:`Settings → Ticket </settings/ticket>`.
+
 Sender Format Seperator: Default value ``via``
-   This is a can be a String you can freely choose. It divided the agents name and the display name of the channel when ever needed.
+   This is a can be a String you can freely choose. It divided the agents name
```

Comment:
It divides the agent's name and the display name of the channel whenever needed.